### PR TITLE
sailpoint

### PR DESCRIPTION
**Pull request by MariePMcGarry**
## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/integrations/security-threat-detection/sailpoint.md [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+description: Install the Sumo Logic app for SailPoint.
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+<head>
+  <meta name="robots" content="noindex" />
+</head>
+
+SailPoint is an identity management solution that helps organizations manage employee permissions, digital identities, information security, data access, and compliance.  The Sumo Logic App for SailPoint helps you monitor the user events, actions, operations, failed logins, successful logins, and user activities to your applications through SailPoint. The App consists of dashboards that give you visibility into the source deletion, user events, and geo locations of authentication events.
```
Comment (kimsauce) [inline docs/integrations/security-threat-detection/sailpoint.md:14]:
```suggestion
SailPoint is an identity management solution that helps organizations manage employee permissions, digital identities, information security, data access, and compliance. The Sumo Logic App for SailPoint helps you monitor the user events, actions, operations, failed logins, successful logins, and user activities to your applications through SailPoint. The App consists of dashboards that give you visibility into the source deletion, user events, and geo locations of authentication events.
```

File: docs/integrations/security-threat-detection/sailpoint.md [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+| count by event_type
+| sort by _count
+```
+
+## Set up collection {#set-up-collection}
+
+Follow the instructions for setting up [Cloud to Cloud Integration for SailPoint App](https://help.sumologic.com/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/sailpoint-source/) to create the source and use the same source category while installing the app.
+
+
+## Installing the App[​](https://help.sumologic.com/docs/integrations/databases/couchbase#installing-the-couchbase-app) {#installing-the-app​}
```
Comment (kimsauce) [inline docs/integrations/security-threat-detection/sailpoint.md:121]: header has a link?

File: docs/integrations/security-threat-detection/sailpoint.md [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+1. From the App Catalog, search for and select the app.
+2. Select the version of the service you're using and click Add to Library. Version selection is applicable only to a few apps currently. For more information, see [Install the Apps from the Library](docs/get-started/apps-integrations.md/#install-apps-from-the-library)
+3. To install the app, complete the following fields.
+    1. App Name. You can retain the existing name, or enter a name of your choice for the app.

+    2. Data Source. Select either of these options for the data source.

+        * Choose Source Category, and select a source category from the list.

+        * Choose Enter a Custom Data Filter, and enter a custom source category beginning with an underscore. Example: (_sourceCategory=MyCategory).
```
Comment (kimsauce) [inline docs/integrations/security-threat-detection/sailpoint.md:134]:
```suggestion
        * Choose Enter a Custom Data Filter, and enter a custom source category beginning with an underscore. Example: (`_sourceCategory=MyCategory`). 
```

File: docs/integrations/security-threat-detection/sailpoint.md [hunk 4]
```diff
@@ -0,0 +1,170 @@
...
+```
+_sourceCategory=Labs/sailpoint
+| json field=_raw "created", "type", "technicalName", "status","operation","actor.name", "action", "name", "target.name", "attributes.sourceName" as created, event_type, technical_name_in_search, event_status, operation, user_name, action, event_desc, target_name, source_name | json "org" as org
+| count by event_type
+| sort by _count
+```
+
+## Set up collection {#set-up-collection}
+
+Follow the instructions for setting up [Cloud to Cloud Integration for SailPoint App](https://help.sumologic.com/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/sailpoint-source/) to create the source and use the same source category while installing the app.
```
Comment (kimsauce) [inline docs/integrations/security-threat-detection/sailpoint.md:118]:
```suggestion
Follow the instructions for setting up [Cloud to Cloud Integration for SailPoint App](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/sailpoint-source) to create the source and use the same source category while installing the app.
```

File: docs/integrations/security-threat-detection/sailpoint.md [hunk 5]
```diff
@@ -0,0 +1,170 @@
...
+```
+
+
+
+#### Sample Queries
+
+Authentication Event
+
+
+```
```
Comment (kimsauce) [inline docs/integrations/security-threat-detection/sailpoint.md:93]:
```suggestion
```sql
```

File: docs/integrations/security-threat-detection/sailpoint.md [hunk 6]
```diff
@@ -0,0 +1,170 @@
...
+| where country_name matches "*" and city matches "*"
+| count by latitude, longitude, country_code, country_name, region, city, postal_code
+| sort _count
+```
+
+
+SailPoint Event Type
+
+
+```
```
Comment (kimsauce) [inline docs/integrations/security-threat-detection/sailpoint.md:109]:
```suggestion
```sql
```

File: docs/integrations/security-threat-detection/sailpoint.md [hunk 7]
```diff
@@ -0,0 +1,170 @@
...
+_sourceCategory=Labs/sailpoint
+| json field=_raw "created", "type", "technicalName", "status","operation","actor.name", "action", "name", "target.name", "attributes.sourceName" as created, event_type, technical_name_in_search, event_status, operation, user_name, action, event_desc, target_name, source_name | json "org" as org
+| count by event_type
+| sort by _count
+```
+
+## Set up collection {#set-up-collection}
```
Comment (kimsauce) [inline docs/integrations/security-threat-detection/sailpoint.md:116]: was this intentional? `{#set-up-collection}`

File: docs/integrations/security-threat-detection/sailpoint.md (hunk 2 above)
Comment (MariePMcGarry) [reply docs/integrations/security-threat-detection/sailpoint.md]: Nope, it should have. I fixed in file

**Review by kimsauce — CHANGES_REQUESTED** [review]
some edits

File: docs/integrations/security-threat-detection/sailpoint.md (hunk 7 above)
Comment (MariePMcGarry) [reply docs/integrations/security-threat-detection/sailpoint.md]: Nope, I edited file and fixed.

File: docs/integrations/security-threat-detection/sailpoint.md [hunk 8]
```diff
@@ -0,0 +1,170 @@
...
+name:"Request Authentication Passed",
+synced:"2022-10-05T11:43:02.214Z",
+_type:"event",
+_version:"v7"
+}
+```
+
+
+
+#### Sample Queries
```
Comment (kimsauce) [inline docs/integrations/security-threat-detection/sailpoint.md:88]:
```suggestion
### Sample Queries
```

**Review by kimsauce — APPROVED** [review]

**Pull request merged**